lib/ProxySQL_Statistics.cpp

```diff
@@ -644,7 +644,11 @@ void ProxySQL_Statistics::system_memory_sets() {
 
 void ProxySQL_Statistics::MyHGM_Handler_sets(SQLite3_result *resultset1, SQLite3_result *resultset2) {
 	MyHGM_Handler_sets_v1(resultset1);
+#ifdef DEBUG
+	if (resultset2) {
```

Comment:
We should add a comment that mentions we enable this feature in debug mode even if web interface is not enabled, for debugging and testing purpose.
Without comment we will forget (especially me).